File: src/main/kotlin/lab/loop/lms/domain/post/service/PostService.java
package lab.loop.lms.domain.post.service;

import lab.loop.lms.domain.post.repository.PostRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Service;

@RequiredArgsConstructor
@Service
public class PostService {
    private final PostRepository postRepository;
}

File: src/main/kotlin/lab/loop/lms/domain/post/service/PostService.kt
package lab.loop.lms.domain.post.service

import lab.loop.lms.domain.post.repository.PostRepository
import lombok.RequiredArgsConstructor
import org.springframework.stereotype.Service

@RequiredArgsConstructor
@Service
class PostService {
    private val postRepository: PostRepository? = null
}
